packages/docs-sidenav/utils/flag-active-nodes.ts

```diff
@@ -1,13 +1,26 @@
-function addIsActiveToNodes(navNodes, currentPath, pathname) {
+import type { NavNodeWithInjectedFields } from '../types'
+
+/**
+ * Injects the `__isActive?: boolean` property into each node
+ */
+function addIsActiveToNodes(
+  navNodes: NavNodeWithInjectedFields[],
+  currentPath: string,
+  pathname: string | null
+) {
   return navNodes
     .slice()
     .map((node) => addIsActiveToNode(node, currentPath, pathname))
 }
 
-function addIsActiveToNode(navNode, currentPath, pathname) {
+function addIsActiveToNode(
+  navNode: NavNodeWithInjectedFields,
```

Comment:
Should this be `NavNode`?

Reply:
Yes 😅
The cyclic calling of `addIsActiveToNodes` & `addIsActiveToNode` was confusing when determining the input types

Reply:
Fixed!